### PR TITLE
Adding necessary metadata for new modules

### PR DESCRIPTION
**Pull request by tkatsoulas**
These metadata are vital for new collectors to appear in the nightly docs

### CODE REVIEW

File: modules/nvme/README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ title: "NVMe devices monitoring with Netdata"
 custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/nvme/README.md
 description: "Monitors NVMe devices health metrics using the nvme CLI tool."
 sidebar_label: "NVMe devices"
+learn_status: "Published"
+learn_topic_type: "References"
+learn_rel_path: "References/Collectors references/Devices"
```
Comment (ilyam8) [inline modules/nvme/README.md:8]: Devices?

Comment (tkatsoulas) [reply modules/nvme/README.md:8]: Yes the category/family of the collector

Comment (tkatsoulas) [reply modules/nvme/README.md:8]: Changing it to _Storage_

Comment (ilyam8) [reply modules/nvme/README.md:8]: Ok, I find it too generic.

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**